### PR TITLE
Add "archive" feature to JSON RPC Providers

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
Add "archive" feature to RPC capabilities, this is needed for some subgraphs and is recently supported by the relay due to the implementation of historical blocks

I was added for `testnet` default and for all other environment overrides such as `previewnet`, `testnet` and `mainnet` overrides.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-the-graph/values-overrides/values-mainnet.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ index-node:
     chains:
       mainnet:
         providers:
-          - features: []
+          - features: ["archive"]
```
Comment (Nana-EC) [inline charts/hedera-the-graph/values-overrides/values-mainnet.yaml:6]: Can we add a README docs for configurations and features to capture this.
It's okay to link to the graph docs also

**Review by Nana-EC — APPROVED** [review]
LG, suggestion to improve documentation

File: charts/hedera-the-graph/values-overrides/values-mainnet.yaml (hunk 1 above)
Comment (AlfredoG87) [reply charts/hedera-the-graph/values-overrides/values-mainnet.yaml:6]: Done! ✅

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**